utils/checkLogIds.py

```diff
@@ -5,71 +5,17 @@
 
 from enum import Enum
 
-class Operation(Enum):
-    READ = 1
-    MODIFY = 2
-
 class Status(Enum):
     OK = 0
-    OK_REPLACED = 1
-    FAIL = 2
-
-def is_good(status):
-    return status in (Status.OK, Status.OK_REPLACED)
+    FAIL = 1
```

Comment:
`Status` is no longer relevant in the new code. Not that it was more relevant in the old one anyway.